configuration/messageproperties/messages_zl_en.properties

```diff
@@ -32,6 +32,7 @@ zl.days=Days
 
 zl.dispensingMustBeWithinOneYearOfEncounter=Next dispensing date has to be within a year of the encounter date
 zl.pleaseSelectADrug=Please select a drug
+zl.select.drug.qty=Make sure each selected drug has a corresponding quantity entered
```

Comment:
@cioan / @mogoodrich - what is the plan for getting corresponding French translations added?  Is there a process for identifying and doing these?

Reply:
Good point @mseaton ... @cioan can we flag @dmdesimone  on this?

Reply:
just added these to the list in https://pihemr.atlassian.net/browse/UHM-6027.